app.js

```diff
@@ -6,68 +6,78 @@ const headEl = document.getElementById('head');
 const bottomEl = document.getElementById('bottom');
 const reportEl = document.getElementById('report');
-const catchphrasesEl = document.getElementById('catchphrases');
-const catchphraseInput = document.getElementById('catchphrase-input');
+const catchphraseEl = document.getElementById('catchphrases');
+const catchphraseInputEl = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let middleCount = 0;
+let bottomCount = 0;
+let catchphrases = [];
 // set state for all of the character's catchphrases
 
 headDropdown.addEventListener('change', () => {
     // get the value of the head dropdown
-
+    const headValue = headDropdown.value;
     // increment the head change count state
-    
-    // update the dom for the head (use style.backgroundImage on the headEl div instead of trying to set the .src -- it's NOT an img tag!)
+    headCount++;
 
+    headEl.style.backgroundImage = `url(./assets/${headValue}-head.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
 });
 
-
 middleDropdown.addEventListener('change', () => {
     // get the value of the middle dropdown
-
+    const middleValue = middleDropdown.value;
     // increment the middle change count state
-    
-    // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    middleCount++;
+    middleEl.style.backgroundImage = `url(./assets/${middleValue}-middle.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
 });
 
-
 bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
-
+    const bottomValue = bottomDropdown.value;
     // increment the bottom change count state
-    
-    // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    bottomCount++;
+    bottomEl.style.backgroundImage = `url(./assets/${bottomValue}-pants.png)`;
     // update the stats to show the new count (call displayStats() to do this work)
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
-    
+
     // push the new catchphrase to the catchphrase array in state
+    catchphrases.push(catchphraseInputEl.value);
+    catchphraseInputEl.value = '';
+    displayCatchphrases();
+});
 
-    // clear out the form input's value so it's empty to the user
-   
-    // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
+// clear out the form input's value so it's empty to the user
 
-});
+// update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
 
 function displayStats() {
```

Comment:
The rubric is asking for the stats to be displayed in a string. We can use a template literal to do this here. We can set the reportEl textContent equal to a template literal in which we call the state of each head, middle, and bottom.